package.json

```diff
@@ -15,6 +16,7 @@
         "babylonjs": "^4.2.0",
         "babylonjs-loaders": "^4.2.0",
         "core-js": "^3.6.5",
+        "jquery": "^3.6.0",
```

Comment:
It would be nice if we could avoid using jQuery, I think. It's not really needed to solve browser compatibility issues these days.